test/integration/MultiIssueanceTokenClass.spec.ts

```diff
@@ -0,0 +1,298 @@
+        this.timeout(50000)
+
+        const symbol = 'RLAU2'
+        const name = 'RLAU2 test'
+        const description = 'my token description'
+        const granularity = 2
+        const amount = 20000000
+        const iconUrl = 'http://a.b.com'
+
+        new RadixTransactionBuilder().createTokenMultiIssuance(
+            identity1.account,
+            name,
+            symbol,
+            description,
+            granularity,
+            amount,
+            iconUrl,
+        )
+        .signAndSubmit(identity1)
+        .subscribe({
+            complete: () => done(),
+            error: e => done(new Error(JSON.stringify(e))),
+        })
+    })
+
+    it('should succeed transacting within granularity', function(done) {
+        this.timeout(50000)
+
+        try {
+            RadixTransactionBuilder.createTransferAtom(
+                identity1.account,
+                identity2.account,
+                RLAU2_URI,
+                new Decimal(100),
+                'test',
+            )
+                .signAndSubmit(identity1)
+                .subscribe({
+                complete: () => {
+                    identity2.account.transferSystem.getAllTransactions().subscribe((txUpdate) => {
+                        if (txUpdate.transaction.message === 'test') {
+                            done()
+                        }
+                    })
+                },
+                error: e =>  done(new Error('This transaction should have been accepted')),
+                })
+        } catch (error) {
+            done()
+        }
+    })
+
+    it('should fail when transacting with the wrong granularity', function(done) {
+        this.timeout(50000)
+
+        try {
+            RadixTransactionBuilder.createTransferAtom(
+            identity1.account,
+            identity2.account,
+            RLAU2_URI,
+            new Decimal(1),
+            )
+            .signAndSubmit(identity1)
+            .subscribe({
+                complete: () => done(new Error('This transaction should have been rejected')),
+                error: e => done(),
+            })
+        } catch (error) {
+            done()
+        }
+    })
+
+    it('should fail creating token with invalid icon url', function(done) {
+        this.timeout(50000)
+
+        const symbol = 'RLAU0'
+        const name = 'RLAU0 test'
+        const description = 'my token description'
+        const granularity = 1
+        const amount = 100000000
+        const iconUrl = 'asdfg'
+
+        new RadixTransactionBuilder().createTokenMultiIssuance(
+            identity1.account,
+            name,
+            symbol,
+            description,
+            granularity,
+            amount,
+            iconUrl,
+        )
+        .signAndSubmit(identity1)
+        .subscribe({
+            complete: () => done(new Error("This token shouldn't be created")),
+            error: () => done(),
+        })
+    })
+
```

Comment:
Maybe add a test when *Alice* tries to `mint`/`burn` a token for which she lacks permissions, maybe token created by *Bob* with permission `TOKEN_OWNER_ONLY`? :)

Reply:
Good idea